src/dbnode/integration/disk_snapshot_test.go

```diff
@@ -172,6 +172,9 @@ func TestDiskSnapshotSimple(t *testing.T) {
 		log.Info("waiting for old snapshot files to be deleted")
 		for _, shard := range shardSet.All() {
 			waitUntil(func() bool {
+				// Increase the time each check to ensure that the filesystem processes are able to progress (some
+				// of them throttle themselves based on time elapsed since the previous time.)
```

Comment:
Would love to have the time to investigate how difficult implementing direct IO would be.  I think it would definitely outweigh further investment in our throttling (which causes all sorts of problems, like this too).

Reply:
Agree'd direct I/O would be cool if we had time for it.

I'm not really adding any additional throttling in this P.R though